app/models/ssh_execution_provider.rb

```diff
@@ -64,3 +64,6 @@ def ssh_port(host)
     end
   end
 end
+
+class ScriptExecutionProvider < SSHExecutionProvider
```

Comment:
don't we want to alias that instead and ideally rename to `ScriptExecutionProvider` and make the SSHExecutionProvider an alias?
```suggestion
SSHExecutionProvider = ScriptExecutionProvider
```
if there are concerns for doing it in this PR, then I'd do it the other way arround, but not subclassing, unless we really want to subclass for some reason.